ContextApp/src/main/java/com/github/migi_1/ContextApp/HelloActivity.java

```diff
@@ -21,11 +31,82 @@ public HelloActivity(){
 //        // Enable verbose logging
 ////        eglConfigVerboseLogging = false;
 //        // Choose screen orientation
-////        screenOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE;
-//        // Enable MouseEvents being generated from TouchEvents (default = true)
+          // Enable MouseEvents being generated from TouchEvents (default = true)
 //        mouseEventsEnabled = true;
 //        // Set the default logging level (default=Level.INFO, Level.ALL=All Debug Info)
-//        LogManager.getLogManager().getLogger("").setLevel(Level.INFO);
+        LogManager.getLogManager().getLogger("").setLevel(Level.INFO);
+        
+    }
+
+         @Override  
+    protected void onResume()  
+    {  
+        super.onResume();
+        
+        //register the lister for the accelerometer
+        mSensorManager.registerListener(this, mSensorManager.getDefaultSensor(Sensor.TYPE_ACCELEROMETER),SensorManager.SENSOR_DELAY_FASTEST);
+    }
+         
+    @Override  
+    protected void onStop()  
+    {  
+        //unregister the sensor listener  
+        mSensorManager.unregisterListener(this);  
+        super.onStop();  
+    } 
+    
+        
+        @Override  
+    public void onCreate(Bundle savedInstanceState)  
+    {  
+        super.onCreate(savedInstanceState);
+        
+        //instantiate the application
+        application = (Main ) getJmeApplication();
+        
+        //start the sensor manager
+        mSensorManager = (SensorManager) getSystemService(SENSOR_SERVICE); 
     }
 
+    @Override
+    public void onSensorChanged(SensorEvent se) {
+        
+        //check whether the game has already been instantiated
+        if (application == null) {
+
+        return;
+
+        }
```

Comment:
Layout is a little bit off in this if statement (two unnecesarry newlines and a tab/space is missing in front of the return statement).